node_modules/col-core/lib/server.js

```diff
@@ -17,14 +17,13 @@ var _ = require('lodash');
 var bodyParser = require('body-parser');
 var config = require('config');
 var cookieParser = require('cookie-parser');
+var cors = require('cors');
```

Comment:
That said though, I haven't looked at what the module does, but if it's just adding headers this would be trivial with Apache